tools/accuracy_checker/configs/opencv/densenet.yml

```diff
@@ -0,0 +1,227 @@
+models:
+  - name: Densenet-121
+
+    launchers:
+      - framework: opencv
+        device: CPU
+        tags:
+          - FP32
+        model:   public/densenet-121/densenet-121.prototxt
+        weights: public/densenet-121/densenet-121.caffemodel
+        inputs:
+            - name: "data"
+              type: INPUT
+              shape: 3, 224, 224
+        adapter: classification
+
+      - framework: opencv
+        device: GPU
+        tags:
+          - FP32
+        model:   public/densenet-121/densenet-121.prototxt
+        weights: public/densenet-121/densenet-121.caffemodel
+        backend: OCV
+        inputs:
+          - name: "data"
+            type: INPUT
+            shape: 3, 224, 224
+        adapter: classification
+
+      - framework: opencv
+        device: GPU
+        tags:
+          - FP16
+        model:   public/densenet-121/densenet-121.prototxt
+        weights: public/densenet-121/densenet-121.caffemodel
+        backend: OCV
+        inputs:
+          - name: "data"
+            type: INPUT
+            shape: 3, 224, 224
+        adapter: classification
+
+    datasets:
+      - name: imagenet_1000_classes
+        data_source: ImageNet
+
+        preprocessing:
+          - type: resize
+            size: 256
+
+          - type: crop
+            size: 224
+
+          - type: normalization
+            mean: 103.94, 116.78, 123.68
+            std:  58.8235294
+
+
+  - name: Densenet-161
```

Comment:
I would suggest to separate configs per model, not model family.
Firstly, it is easy to run specific model.
Secondly, it can be paralleed on several machines. Some public models validation on full datasets are terribly slow. You risks to get timeout break.

Reply:
@eaidova done